maintainers/maintainer-list.nix

```diff
@@ -12638,6 +12638,11 @@
     github = "lenivaya";
     githubId = 49302467;
   };
+  lennartrth = {
+    name = "Lennart rth";
+    github = "lennart-rth";
+    githubId = 80266866;
+  };
```

Comment:
Please separate this into another commit *preceding* the main one, titled `maintainers: add lennart-rth`